src/lib/types/MessageUpdate.ts

```diff
@@ -59,9 +59,8 @@ export interface MessageWebSearchSourcesUpdate
 	sources: WebSearchSource[];
 }
 export interface MessageWebSearchFinishedUpdate
-	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished> {
-	webSearch: WebSearch;
-}
+	extends BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished> {}
```

Comment:
```suggestion
export type MessageWebSearchFinishedUpdate = BaseMessageWebSearchUpdate<MessageWebSearchUpdateType.Finished>
```

Reply:
handled in https://github.com/huggingface/chat-ui/pull/1179/commits/154d896bc4e56b44301cccb1375f70e4c525fb8f